src/renderer/features/multisig-wallet-create/flexible-multisig/model/assign-model.ts

```diff
@@ -0,0 +1,212 @@
+import { type ApiPromise } from '@polkadot/api';
+import { attach, combine, createEvent, createStore, sample } from 'effector';
+import { sortBy } from 'lodash';
+import { spread } from 'patronum';
+
+import { AccountType, CryptoType, type NoID, ProxyVariant, SigningType, WalletType } from '@/shared/core';
+import { type FlexibleMultisigAccount, type FlexibleProxiedAccount } from '@/shared/core/types/account';
+import { Step, nonNullable, nullable, toAccountId } from '@/shared/lib/utils';
+import { createComplexTxStore } from '@/shared/transactions';
+import { accounts } from '@/domains/network';
+import { networkUtils } from '@/entities/network';
+import { transactionBuilder } from '@/entities/transaction';
+import { walletModel } from '@/entities/wallet';
+import { signModel } from '@/features/operations/OperationSign';
+import { submitModel, submitUtils } from '@/features/operations/OperationSubmit';
+
+import { confirmModel } from './confirm-model';
+import { flexibleMultisigFeature } from './feature';
+import { flexibleMultisigModel } from './flexible-multisig-create';
+import { formModel } from './form-model';
+import { signatoryModel } from './signatory-model';
+import { walletProviderModel } from './wallet-provider-model';
+
+const $api = combine(flexibleMultisigFeature.state, (state): ApiPromise | null => {
+  if (state.status !== 'running') return null;
+  return state.data.api;
+});
+
+const $confirmMeta = confirmModel.$confirmMap.map(s => (nonNullable(s[0]) ? s[0].meta : null));
+const $signer = $confirmMeta.map(m => (nonNullable(m) ? m.signatory : null));
+
+const $coreTx = combine(
+  {
+    meta: $confirmMeta,
+    signatories: signatoryModel.$signatories,
+    proxyAddress: confirmModel.$proxyAddress,
```

Comment:
confirmModel usage if transaction creation flow confuses me a bit, let's redo this part